Code/GraphMol/MolDraw2D/CMakeLists.txt

```diff
@@ -12,45 +12,40 @@ rdkit_headers(MolDraw2D.h
   DEST GraphMol/MolDraw2D
 )
 
+rdkit_library(MolDraw2D MolDraw2D.cpp MolDraw2DSVG.cpp
+  MolDraw2DDetails.cpp MolDraw2DUtils.cpp
+  LINK_LIBRARIES
+  FileParsers SmilesParse Depictor MolTransforms RDGeometryLib
+  RDGeneral SubstructMatch Subgraphs GraphMol EigenSolvers
+RDGeometryLib
+  ${RDKit_THREAD_LIBS} )
+
 if(RDK_BUILD_QT_SUPPORT)
   #find_package(Qt5 COMPONENTS Widgets OpenGL REQUIRED)
   find_package(Qt5Widgets REQUIRED)
   find_package(Qt5OpenGL REQUIRED)
 
-  include_directories(
-    ${Qt5Widgets_INCLUDE_DIRS}
-    ${Qt5OpenGL_INCLUDE_DIRS}
-    )
-
-  set(QTDRAW2D_SRC MolDraw2DQt.cpp)
-  set(EXTRA_LOCAL_LIBS ${EXTRA_LOCAL_LIBS} ${Qt5Widgets_LIBRARIES}  ${Qt5OpenGL_LIBRARIES} )
+  target_sources(MolDraw2D PRIVATE MolDraw2DQt.cpp)
+  target_include_directories(MolDraw2D PUBLIC ${Qt5Widgets_INCLUDE_DIRS} ${Qt5OpenGL_INCLUDE_DIRS})
+  target_compile_options(MolDraw2D PRIVATE "-std=c++11")
```

Comment:
The c++ standard library can't be set here as this is a global setting and may actually be libstdc++